README.md

```diff
@@ -96,31 +96,52 @@ Optional high-code extensions allow almost infinite customization in a modular w
 <img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/dashboard_examples.png" width="1300"/>
 </p>
 
-### Live demo
+## Packages
```

Comment:
@huong-li-nguyen  here we apply the packages tables to input all packages

Reply:
I like the changes, and also the table. The only change I would do still is:
- move the table to the top under the icons already present there
- add the package specific icons to the table
- add the relevant links per package to the table
- finally add a link to the packages section

Since the table is just two lines it will not be very distracting, but will give an immediate overview. And we still start mainly with Vizro Core just below

Reply:
> ..
Hey Max, can we clarify some points below:
- move the table to the top under the icons already present there
: for this one, the top before even what is Vizro or more specifically which section.
- About the icons, do we have icons for vizro-ai or i misunderstood something here

Reply:
![Screenshot 2024-02-14 at 22 00 59](https://github.com/mckinsey/vizro/assets/90609403/a489bd60-7559-423a-9d5d-fc11c3401492)

I think what he means is moving the table below the current vizro logo, because the badges below that logo currently refer to vizro-core only. With icons, I think he means the badges  (so python versions, docs passing, etc.) :D

Reply:
Yes indeed, thanks for that, I wrote the comment a little bit in a hurry :P

Reply:
So we could leave the repo specific badges/icons (like license, openssf) where they are, but move the package specific icons to the table (python, docs, openssf).

And yes, we could create the relevant icons also for `vizro-ai`.

Wdyt?

Reply:
not sure about creating icons for vziro-ai, probably we should stick with one logo to represent all vizro relevant packages?

Reply:
Just to clarify, with icon/badges I mean this: 
![image](https://github.com/mckinsey/vizro/assets/83698606/1d4c01d8-5c4e-4af6-8385-5fb4e93ff7ef)

Why should we not create them specifically for `vizro-ai`?

The logo
![image](https://github.com/mckinsey/vizro/assets/83698606/31f7ae16-7598-4363-a074-dc3b4f98527c)
should of course remain at the top!

Reply:
oh i see, i thought you were referring a separate log for vizro-ai. 
Those has been placed in [vizro-ai page readme top](https://github.com/mckinsey/vizro/blob/main/vizro-ai/README.md).

If we add all those icons and badge, it will look a lot in side of one table(too much for separate cols and too crowded for one cell.)  if it is fine, can we remove licence /docs passing / passing best practice in tables and let it just be at the top of relevant page

Reply:
I see - how about this:

- leave the licence and best practices at the top
- add the docs passing **instead** of the current link, because clicking the badge gets you to the docs
- add the python version as an additional column
- those three badges/icons should be repeated at the top of the individual folder READMEs

Then put that table below this line
<img width="827" alt="image" src="https://github.com/mckinsey/vizro/assets/83698606/b381bcb3-4dcc-40f6-bf21-311a47af7b76">
with a clickable (small) header "Packages" that redirects you to the "Packages" section further down

Like that we are not crammed, have all the info in the right place, and still attribute the main attention to vizro-core?

Reply:
I think the content of the table all makes sense 👍

placing the table at the top of the page may break the flow of the page for people visiting the page for the first time (especially when a larger number of plugins are added in future, which will increase the height of the table), and people visiting the page for the first time are unlikely to be interested in packages on the their initial view? (whereas repeat viewers are likely to go straight to the relevant package folder in the folder tree above?)

so it might make a smoother route for new/returning viewers if the table is shown near the bottom of the page (under the initial explanation of the Vizro core package - which also removes the need to repeat the explanation of the vizro-core package below that)?